clash-lib/src/Clash/Normalize/Primitives.hs-boot

```diff
@@ -0,0 +1,6 @@
+module Clash.Normalize.Primitives where
+
+import Clash.Core.Term (PrimInfo)
+
+undefined :: PrimInfo
+undefinedX :: PrimInfo
```

Comment:
Is it really worth it to add a `.hs-boot` file just to get the name of these two functions? I'd be inclined to make an exception for these and just refer to them as a string, in the interest of compilation speed.

Reply:
I'd rather keep everything uniform as TH names.